wagtail/admin/templates/wagtailadmin/pages/listing/_locked_indicator.html

```diff
@@ -7,6 +7,5 @@
 {% endcomment %}
 
 {% if page.locked %}
-    {# TODO: Different look when it's locked by current user? #}
-    <span class="indicator locked-indicator icon icon-locked" title="{% trans 'This page is locked to further editing' %}"></span>
+    <span class="indicator locked-indicator {% if page.owner == request.user %}locked-indicator-owner{% endif %} icon icon-locked" title="{% trans 'This page is locked to further editing' %}"></span>
```

Comment:
It is important that anything visual is also communicated to screen readers/ assistive technology. 

Please ensure that the translation in the title also differs when it is the current user. 

Maybe 
`{% trans 'This page has been locked to further editing by you' %}`

Remember that you will need to use two full trans strings and not build the bits dynamically.